pkg/probe/vpn_ssl.go

```diff
@@ -3,33 +3,59 @@ package probe
 import (
 	"log"
 
+	"github.com/bluecmd/fortigate_exporter/internal/config"
 	"github.com/bluecmd/fortigate_exporter/pkg/http"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+type VPNUser struct {
+	UserName string `json:"user_name"`
+}
+
+type VPNUsers struct {
+	Results []VPNUser `json:"results"`
+	VDOM    string    `json:"vdom"`
+	Version string    `json:"version"`
+}
+
 func probeVPNSsl(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	savedConfig := config.GetConfig()
```

Comment:
I don't like this very much, but I understand why you went this path.

I sent out https://github.com/bluecmd/fortigate_exporter/pull/125 for another way that I like more.
This would cause this part to become:
```go
import "flag"
...
        flag.Set("max-vpn-users", "10")
        config.MustReInit()
        if !testProbe(probeVPNSsl, c, r) {
                t.Errorf("probeVPNSsl() returned non-success")
        }
```

Reply:
Ready for your usage now :)